### PR TITLE
Release 3.6.0

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-release-3.6.0](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-release-3.6.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.6.0 (2020-05-08)
+
+- Added modern theme styles to the tab dropdown button. [#84](https://github.com/blackbaud/skyux-popovers/pull/84)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]: Is "tab" correct? I see that in the code in the PR, but I don't see "tab" mentioned anywhere in the docs. We just refer to the dropdown button.

Comment (Blackbaud-PaulCrowder) [reply CHANGELOG.md:3]: Yes. There's a type of dropdown button specifically for the tab component.

Comment (blackbaud-johnly) [reply CHANGELOG.md:3]: Sounds good. Just wanted to make sure.

**Pull request merged**